manifests/k/KDE/Kate/21.08.2/KDE.Kate.installer.yaml

```diff
@@ -1,14 +1,12 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
-PackageIdentifier: KDE.Kate
-PackageVersion: 21.08.2
-MinimumOSVersion: 10.0.0.0
-InstallerType: nullsoft
-Scope: machine
+PackageIdentifier: KDE.Kate.Release
+PackageVersion: master-1459
+InstallerType: appx
 UpgradeBehavior: install
 Installers:
 - Architecture: x64
-  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Kate_Release_win64/1456/artifact/kate-21.08.2-1456-windows-msvc2019_64-cl.exe
-  InstallerSha256: D95F20BE05A06358090201A956D608DD0457CC7916A159403F83E671D7F045A7
+  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Kate_Release_win64/lastSuccessfulBuild/artifact/kate-21.08.2-1459-windows-msvc2019_64-cl-sideload.appx
+  InstallerSha256: f8692d5dfd0bc6760926579dcab927d545671cc346126f3b71ea6cd598a8e732
```

Comment:
```
  InstallerSha256: f8692d5dfd0bc6760926579dcab927d545671cc346126f3b71ea6cd598a8e732
```
`InstallerType` is set to `Appx`, so you're also missing `SignatureSha256` and `PackageFamilyName` here.

Reply:
> http://github.com/microsoft/winget-pkgs/pull/33718#r742084881.

Is that information within the package? I did search within it via usage of http://github.com/kde/ark, but I did not observe the presence of that information.

Reply:
`winget hash --msix <PATH_TO_APPX_OR_MSIX>` will give you the `SignatureSha256`

PackageFamilyName can be found through `Get-AppxPackage | Select-Object -Last 1` in PowerShell.